src/autolabel/models/google.py

```diff
@@ -0,0 +1,180 @@
+            "safety_settings": {
+                HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,
+                HarmCategory.HARM_CATEGORY_HATE_SPEECH: HarmBlockThreshold.BLOCK_NONE,
+                HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_NONE,
+                HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,
+            },
+        }
+
+        super().__init__(config, cache)
+        # populate model name
+        self.model_name = config.model_name() or self.DEFAULT_MODEL
+        # populate model params and initialize the LLM
+        model_params = config.model_params()
+        self.model_params = {
+            **model_params,
+            **self.DEFAULT_PARAMS,
+        }
+        if self._engine == "chat":
+            self.llm = ChatGoogleGenerativeAI(
+                model=self.model_name, **self.model_params
+            )
+        else:
+            self.llm = ChatGoogleGenerativeAI(
+                model=self.model_name, **self.model_params
+            )
+
+        self.tiktoken = tiktoken
+
+    @retry(
+        reraise=True,
+        stop=stop_after_attempt(5),
+        wait=wait_exponential(multiplier=1, min=2, max=10),
+        before_sleep=before_sleep_log(logger, logging.WARNING),
+    )
+    def _label_with_retry(self, prompts: List[str]) -> LLMResult:
+        start_time = time()
+        response = self.llm.generate(prompts)
+        return response, time() - start_time
+
+    def _label_individually(self, prompts: List[str]) -> RefuelLLMResult:
+        """Label each prompt individually. Should be used only after trying as a batch first.
+
+        Args:
+            prompts (List[str]): List of prompts to label
+
+        Returns:
+            RefuelLLMResult: RefuelLLMResult object
+        """
+        generations = []
+        errors = []
+        latencies = []
+        for i, prompt in enumerate(prompts):
+            try:
+                response, latency = self._label_with_retry([prompt])
+                for generation in response.generations[0]:
+                    generation.text = generation.text.replace(
+                        self.SEP_REPLACEMENT_TOKEN, "\n"
+                    )
+                generations.append(response.generations[0])
+                errors.append(None)
+                latencies.append(latency)
+            except Exception as e:
+                print(f"Error generating from LLM: {e}, returning empty generation")
+                generations.append([Generation(text="")])
+                errors.append(
+                    LabelingError(
+                        error_type=ErrorType.LLM_PROVIDER_ERROR, error_message=str(e)
+                    )
+                )
+                latencies.append(0)
+
+        return RefuelLLMResult(
+            generations=generations, errors=errors, latencies=latencies
+        )
+
+    def _label(self, prompts: List[str]) -> RefuelLLMResult:
```

Comment:
we don't need to do this for google. Separator replacement token was something done specifically for RefuelLLM in the past. Does not need to be replicated here.

Let's follow models/openai.py